### PR TITLE
Changes to SearchBar and new SearchPage component

### PR DESCRIPTION
**Pull request by leonzh2k**

### CODE REVIEW

**Review by roundband5 — APPROVED** [review]
Looks Good

File: src/modules/Navbar/Searchbar.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.search-container{
+  position:relative;
+}
+  
+.search-container input{
+  display:flex;
```
Comment (lchua2314) [inline src/modules/Navbar/Searchbar.css:6]: Add space

File: src/modules/Navbar/Searchbar.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.search-container{
```
Comment (lchua2314) [inline src/modules/Navbar/Searchbar.css:1]: Add space after "container"

File: src/modules/Navbar/Searchbar.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  justify-content: left;
+  align-items: center;
+  padding: 20px 10px;
+  background: rgb(126, 124, 124);
+  font-size: 17px;
+  border: solid grey;
+  cursor: pointer;
+}
+
+.search-container button{
```
Comment (lchua2314) [inline src/modules/Navbar/Searchbar.css:16]: Add space after "button"

File: src/modules/Navbar/Searchbar.css (hunk 1 above)
Comment (lchua2314) [reply src/modules/Navbar/Searchbar.css:6]: Add space to all the other CSS

File: src/modules/Navbar/Searchbar.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  function handleInputChange(e) {
+    setSearchString(e.target.value)
+  }
+
+  return (
+    <div className="search-container">
+      <input type="text" placeholder="" name="search" onChange={(handleInputChange)} value={searchString} />
+        <a href={"/search?query=" + searchString}>
+          <button type="submit">
+            <img src={SearchIcon} className="Search-Icon" width="40" height="40" alt="search icon (click to search)" />
```
Comment (lchua2314) [inline src/modules/Navbar/Searchbar.js:18]: Move width and height to CSS file

**Review by lchua2314 — CHANGES_REQUESTED** [review]
Fix a bit

**Pull request merged**